sonar-project.properties

```diff
@@ -3,4 +3,5 @@ sonar.projectKey=ministryofjustice_laa-access-civil-legal-aid
 
 # relative paths to source directories. More details and properties are described
 # in https://sonarcloud.io/documentation/project-administration/narrowing-the-focus/
-sonar.sources=src
+sonar.sources=.
+sonar.tests=tests/unit_tests
```

Comment:
Should Sonar Cloud only be looking at the unit tests or the broader test directory?
[https://github.com/ministryofjustice/laa-civil-case-api/blob/main/sonar-project.properties#L7](https://github.com/ministryofjustice/laa-civil-case-api/blob/main/sonar-project.properties#L7)

Reply:
The sonar source needs to look at our whole project and the coverage is just code coverage of unit tests, our functional tests are looking at the service.